crates/resolver/src/caching_client.rs

```diff
@@ -610,8 +610,12 @@ mod tests {
         .unwrap();
```

Comment:
(_Another commit message review comment_)

This commit message could use word-wrap treatment